lib/pages/card_page.dart

```diff
@@ -0,0 +1,103 @@
+import 'package:MobileOne/utility/colors.dart';
+import 'package:MobileOne/widgets/widget_loyaltyCards.dart';
+import 'package:barcode_widget/barcode_widget.dart';
+import 'package:flutter/material.dart';
+
+class Cards extends StatefulWidget {
+  @override
+  CardsState createState() => CardsState();
+}
+
+class CardsState extends State<Cards> {
+  @override
+  Widget build(BuildContext context) {
+    ScreenArguments args = ModalRoute.of(context).settings.arguments;
+    return Scaffold(
+      body: Column(
+        children: <Widget>[
+          Padding(
+            padding: const EdgeInsets.only(top: 30.0),
+            child: Align(
+              alignment: Alignment.topLeft,
+              child: IconButton(
+                icon: Icon(Icons.arrow_back),
+                onPressed: () {
+                  goToLoyaltyCardsPage();
+                },
+              ),
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.only(left: 50.0),
+            child: Stack(
+              children: <Widget>[
+                Container(
+                  height: MediaQuery.of(context).size.height - 150,
+                  width: MediaQuery.of(context).size.width - 128,
```

Comment:
instead of putting hard size like height - 150 try to compute like this: height * .8 => 80%